mlserver/cli/constants.py

```diff
@@ -46,7 +46,13 @@
     ./requirements.tx[t] \\
     .
 
-RUN ./hack/build-env.sh . ./envs/base
+USER root
+# Install dependencies system-wide, to ensure that they are available for every
+# user
+RUN ./hack/build-env.sh . ./envs/base && \
+    chown -R 1000:0 ./envs/base && \\
```

Comment:
is there a reason why we need `chown -R 1000:0 ./envs/base` ?

Reply:
Do you mean besides the `chmod`? Good point - maybe it would be enough with the `chmod`... I just mimicked what we already have in the main `Dockerfile`. I didn't to try my luck too much with permissions :grimacing: